.pre-commit-config.yaml

```diff
@@ -134,6 +133,7 @@ repos:
         name: pylint with optional checks
         args:
           - --rcfile=.pylintrc
+          - --ignore=htmlcov/
```

Comment:
you should not alter this file. If you think you need these changes and they are good for the whole OCA, please propose the change to https://github.com/OCA/oca-addons-repo-template